classes/plugins/LazyLoadPlugin.inc.php

```diff
@@ -78,6 +78,7 @@ function setEnabled($enabled) {
 			$contextId = 0;
 		}
 		$this->updateSetting($contextId, 'enabled', $enabled, 'bool');
+		HookRegistry::call('Plugin::setEnabled', [$this, $enabled]);
```

Comment:
There are some important plugins that don't call `parent::setEnabled` in their own implementations of `setEnabled` -- the ORCID profile plugin and anything extending `BlockPlugin` or `GatewayPlugin`, for example. There are also other uses of `setEnabled` that you might not expect (https://github.com/pkp/customBlockManager/blob/a758bdc116df01e3ac5c2202b4d19bbca3f72e10/controllers/grid/form/CustomBlockForm.inc.php#L112). So I think especially for `stable-3_3_0` it would be beneficial to rely on the `enable` / `disable` grid actions rather than the `Plugin::setEnabled` call.

Reply:
Hmm, really a surprise 😁

Reply:
@asmecher Ok, I've updated this PR to recover the hook.